apps/web/app/[locale]/timesheet/[memberId]/components/EditTaskModal.tsx

```
@@ -0,0 +1,229 @@
+                            handleChange={handleChange}
+                            itemToString={(item, displayKey) => getNestedValue(item, displayKey) || ''}
+                            itemToValue={(item, valueKey) => getNestedValue(item, valueKey) || ''}
+                        />
+                    </div>
+                    <div className=" flex flex-col items-center">
+                        <label className="text-gray-500 mr-6 ">Billable</label>
+                        <div className="flex items-center gap-3">
+                            <div className="flex items-center gap-x-2">
+                                <div
+                                    className={clsxm(`w-6 h-6 flex items-center bg-[#6c57f4b7] rounded-full p-1 cursor-pointer`, 'bg-white')}
+                                    onClick={() => setIsBillable && setIsBillable(!isBillable)}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)